addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8906,7 +9134,12 @@ msgctxt "#19294"
 msgid "Delete this recording permanently?"
 msgstr ""
 
-#empty strings from id 19295 to 19498
+#: system/settings/settings.xml
```

Comment:
Mind adding a contextual note? I suppose it's the description for the timer setting?